presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorSplitManager.java

```diff
@@ -246,7 +251,15 @@ private ConnectorSplit createSplit(BucketShards bucketShards)
                 addresses = ImmutableList.of(node.getHostAndPort());
             }
 
-            return new RaptorSplit(connectorId, shardId, addresses, effectivePredicate, transactionId, columnTypes);
+            return new RaptorSplit(
+                    connectorId,
+                    shardId,
+                    deltaShardUuid.isPresent() ? ImmutableMap.of(shardId, deltaShardUuid.get()) : ImmutableMap.of(),
```

Comment:
`deltaShardUuid.map(uuid -> ImmutableMap.of(shardId, uuid)).orElse(ImmutableMap.of()),`